docs/guides/meta/meta-avatars-setup.md

```diff
@@ -18,16 +19,23 @@ Once Normcore is set up in your project, you'll be able to install the **Normcor
 
 <video width="100%" controls><source src={metaXR} /></video> 
 
+## Install the Meta Avatars SDK sample
+**Normcore Meta XR** depends on scripts and assets that are included in a user sample from the **Meta Avatars SDK** package.
+
+Import this dependency automatically using the pop-up dialog. You can always bring up the dialog again using the menu bar: `MetaAvatarsSDK/Normcore/Check Dependencies`.
+
+<video width="100%" controls><source src={packageDependencies} /></video>
```

Comment:
I don't understand what causes this pop-up to show up "automatically". What package causes this dialog to appear? It's not our Normcore Meta XR package. If it's the "Meta Avatars SDK" package, then we need to show the process of importing that package in the video (and also describe it better in the instructions copy)

Reply:
Oh snap this pop-up /is/ coming from our package. Can we use a different menu bar path then? I don't want to try to bury it within the Meta one in case they change their path.

Also, can we combine this step into the step where you install the UPM package? It will make the guide shorter and I think it will be easier to understand since this happens automatically after installing our package.